3.2/Dockerfile

```diff
@@ -50,6 +50,8 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 		zlib1g-dev \
 		# YJIT dep
 		rustc \
+		# Build gems with Rust extensions dep
+		clang-14 \
```

Comment:
Could you please remove this from the 3.2/Dockerfile and instead place these lines in `Dockerfile.template`?

The next update to ruby will include these updates. After the change, you can run `./shared/gen-dockerfiles.sh 3.2.2` for a Dockerfile you can use